api/PerformanceObserver.json

```diff
@@ -0,0 +1,216 @@
+{
+  "api": {
+    "PerformanceObserver": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PerformanceObserver",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "52"
```

Comment:
The [MDN article](https://developer.mozilla.org/en-US/docs/Web/API/PerformanceObserver#Specifications) also has data on availability in workers for Chrome and Opera (both desktop and mobile). Maybe we can incorporate those information as notes here (it should suffice for the main feature)?

Reply:
Added this as a sub features, as per our conventions: https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#api-specific-subfeatures